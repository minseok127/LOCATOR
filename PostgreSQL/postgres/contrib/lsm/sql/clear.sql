--
-- Delete the extension and server 
--

\c lsmtest

DROP SERVER lsm_server;  
DROP EXTENSION lsm_fdw;  
  
\c postgres  
DROP DATABASE lsmtest;  
