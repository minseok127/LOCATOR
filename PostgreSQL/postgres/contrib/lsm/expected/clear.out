--
-- Delete the extension and server 
--
\c lsmtest
You are now connected to database "lsmtest" as user "postgres".
DROP SERVER lsm_server;  
DROP SERVER
DROP EXTENSION lsm_fdw;  
DROP EXTENSION
  
\c postgres  
You are now connected to database "postgres" as user "postgres".
DROP DATABASE lsmtest;  
DROP DATABASE
